tests/test-usermeta.ts

```diff
@@ -234,6 +234,33 @@ describe("usermeta", function() {
       });
     });
 
+    describe(".getBulk", function() {
+      it("delegates to the jsonClient", function() {
+        usermetaClient.getBulk("username1,username2", ["age"], td.function('callback'));
```

Comment:
`usermetaClient.getBulk` should take an array of usernames and do the `.join` internally (when needed, to build the URI of the external request). It's strange to pass an array as a comma separated string when we're in a language that supports arrays :). Also, if usernames contain comma, we can still make it work - since we can `encodeURIComponent` each individual username, a username with a comma won't create a weird bug.